README.md

```diff
@@ -1,44 +1,89 @@
-# DabApps ESLint Configuration
+# ESLint Config
 
 
 ## About
-This repository includes 3 different configs that inherit from each other.
-
-  - Base
-  - React (extends the `base`)
-  - React Native (extends `react`, `base`)
+This repository includes a collection of eslint configs that can be combined to lint just about any project, and a default config for all projects that follow our recent patterns.
 
 
 ## Install
 
-Install a specific version of the eslint-config in your package.json. 
+Install a specific version of the eslint-config with NPM. You can see a full list of versions [here](https://github.com/dabapps/eslint-config/releases).
+
+```shell
+npm i https://github.com/dabapps/eslint-config.git#v3.0.0 --save-dev
+```
+
+This will update your package.json automatically.
 
 ```json
 "devDependencies": {
-  "eslint-config": "dabapps/eslint-config#2.0.0",
+  "eslint-config": "https://github.com/dabapps/eslint-config.git#v3.0.0",
 }
 ```
 
-Then you have to choose between the different configs, depending on the project type. Currently there are 3 different configs.
 
-  - base
-  - react
-  - react-native
+## Configuration
 
 
-## Running eslint
+### NPM scripts
 
-Add an `.eslintrc` file in the root of your project that extends a particular config.
+Add the following scripts to your package.json
+
+```json
+{
+  "scripts": {
+    "lint": "npm run lint-src && npm run lint-tests",
+    "lint-src": "eslint -c .eslintrc.json src/",
+    "lint-tests": "eslint -c .eslintrc-test.json tests/"
+  }
+}
+```
+
+
+### Default config
+
+Create an `.eslintrc.json` in the route of the project. For most projects you can use the default config like so:
+
+```json
+{
+  "extends": [
+    "eslint-config"
+  ]
+}
+```
 
-    {
-      "extends": "eslint-config-dabapps/base/.eslintrc", // For Base
-      "extends": "eslint-config-dabapps/react/.eslintrc", // For React
-      "extends": "eslint-config-dabapps/react-native/.eslintrc", // For React-Native
-    }
+Explanation of the default config below
 
-It's recommeneded to then run this in your test script e.g.
 
-    "scripts": {
-      "eslint": "eslint 'src/js/'",
-      "test": "npm run eslint && npm run jest",
-    }
+### Custom config
+
+But if you want to customise the project because, for example, this project is using a different ECMA version, module system, or framework, you can use a combination of the other configs available in this module. To replicate the default config for example, you could use the following:
+
+```json
+{
+  "extends": [
+    "eslint-config/commonjs",
+    "eslint-config/browser",
```

Comment:
This is fine. Someone has actually published `eslint-config` to npm to contain their config (which really annoys me), but doing it like this is perfectly fine, provided this is also the value inside the package.json. NPM uses the value from the package.json, no matter what key you use inside the projects package.json (something I found out the hard way)